zap/src/main/java/org/parosproxy/paros/core/scanner/ScannerParam.java

```diff
@@ -311,10 +311,17 @@ protected void parse() {
         }
     }
 
-    private void removeOldOptions() {
-        final String oldKey = "scanner.deleteOnShutdown";
-        if (getConfig().containsKey(oldKey)) {
-            getConfig().clearProperty(oldKey);
```

Comment:
Keep as is and just drop the final, use the same variable to fix the typo.

Reply:
done